install.php

```diff
@@ -207,6 +250,186 @@ public function installOrUpdate($parent)
 	}
 
 	/**
+	 * Method to process SQL updates previous to the install process
+	 *
+	 * @param   object  $parent  Class calling this method
+	 *
+	 * @return  boolean          True on success
+	 */
+	public function preprocessUpdates($parent)
```

Comment:
and will it run update files twice then? where are you updating the version so it cannot run it twice?